unions/errorsAsData/README.md

```diff
@@ -0,0 +1,33 @@
+## Unions: Errors as Data
+
+This example shows how to use a union type to represent errors as data. This is a common pattern when using GraphQL to represent errors in a way that is easy to consume by the client.
+
+## Getting Started
+
+[Install](https://stepzen.com/docs/quick-start/install-and-setup) the StepZen command line interface.
+
+To run these examples,
+
+- `git clone` this repo
+- Change to the right working directory.
+- run `stepzen start`
+
+## Explore
+
+You can use the following query to explore the data:
+
+```graphql
+mutation {
+  auth(email: "demo", password: "demo") {
+    ... on Token {
```

Comment:
probably want to include `__typename` to allow the client to determine what is being returned.